modules/flowable-engine/src/main/java/org/flowable/engine/impl/persistence/entity/ExecutionEntityImpl.java

```diff
@@ -845,7 +845,7 @@ protected VariableInstanceEntity createVariableInstance(String variableName, Obj
         variableListenerSession.addVariableData(variableInstance.getName(), VariableListenerSessionData.VARIABLE_CREATE, 
                 variableInstance.getProcessInstanceId(), ScopeTypes.BPMN, variableInstance.getProcessDefinitionId());
         
-        Clock clock = CommandContextUtil.getProcessEngineConfiguration().getClock();
+            Clock clock = CommandContextUtil.getProcessEngineConfiguration().getClock();
```

Comment:
Nit pick: I don't think this indentation is needed and needs to be removed.

Reply:
Fixed.